web/src/components/molecules/Content/Form/index.tsx

```diff
@@ -68,6 +91,10 @@ const ContentForm: React.FC<Props> = ({
     }
   }, [form, model?.schema.fields, model?.schema.id, itemId, onItemCreate, onItemUpdate]);
 
+  const handleLink = (_asset: Asset) => {
+    // TODO: implement link asset with content
```

Comment:
Does this mean this PR is just the UI?

Reply:
In this PR I connected Asset with Content since I developed Asset in the first place. and it seemed a bit confusing for Basel to Connect it.
So me and Basel decided that I will connect Asset with Content and Basel will handle the "Link Asset" part.

Reply:
In this PR I connected `asset` with `content` since I developed asset in the first place and it seemed a bit confusing to Basel to connect it.
So, me and Basel have decided that I connect `asset` with `content` and he will handle the `link` part.